card/src/main/java/com/adyen/checkout/card/ui/AddressFormInput.kt

```diff
@@ -46,6 +46,8 @@ class AddressFormInput @JvmOverloads constructor(
 
     private lateinit var delegate: AddressDelegate
 
+    private var currentSpec: AddressSpecification = AddressSpecification.fromString(null)
```

Comment:
`currentSpec` should be null initially because before we get any update from output data we don't populate the view and for that state it should be null.